contracts/pool/pools/liquid/variants/ProlongedLiquidTokenLockingPool.sol

```diff
@@ -3,21 +3,23 @@ pragma solidity 0.8.18;
 
 import "../LiquidTokenLockingPoolBase.sol";
 
-error LockPeriodNotFinished();
-
 /**
  * @title ProlongedLiquidTokenLockingPool
  * @dev Provides functionality for prolonged locking of tokens with a lock period
  */
-contract ProlongedLiquidTokenLockingPool is LiquidTokenLockingPoolBase {
+abstract contract ProlongedLiquidTokenLockingPool is
+    LiquidTokenLockingPoolBase
+{
+    error LockPeriodNotFinished();
```

Comment:
Mozes li prebacit ovaj error ispod mappinga, cisto da postujemo [solidity style guide](https://docs.soliditylang.org/en/latest/style-guide.html#order-of-layout)